src/dotnet/commands/dotnet-new/CSharp_Mstest/$projectName$.csproj

```diff
@@ -1,12 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp1.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20170106-08" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20170123-02" />
```

Comment:
We have a pending PR to remove the old new verb and switch to using new3 as the new verb. You should sync with @mlorbetske to make sure this version is updated there.